src/styles/blocks/main-con.scss

```
@@ -0,0 +1,3 @@
+.main-content__products {
+  margin: auto;
+}
```

If you only have styles for elements - still use nesting.
Also, name of the file should be exact like the name block.
```suggestion
.main-content {
  &__products {
    margin: auto;
  }
}
```